### PR TITLE
Document WaitForFirstConsumer alternative to per-zone StorageClass

### PR DESCRIPTION
**Pull request by PrismaPhonic**
This PR Updates local lockserver suggestions in generated docs to include a newer and simpler approach.

### CODE REVIEW

File: pkg/apis/planetscale/v2/etcdlockserver_types.go [hunk 1]
```diff
@@ -82,9 +82,10 @@ type EtcdLockserverTemplate struct {
 	// for each etcd instance to store its data files.
 	// This field is required.
 	//
-	// IMPORTANT: For a cell-local lockserver, you must set a storageClassName
-	// here for a StorageClass that's configured to only provision volumes in
-	// the Availability Zone that corresponds to the Vitess cell.
+	// IMPORTANT: For a cell-local lockserver, you have two options. You either should set a storageClassName here for
```
Comment (avaidyanatha) [inline pkg/apis/planetscale/v2/etcdlockserver_types.go:85]:
```suggestion
	// IMPORTANT: For a cell-local lockserver, you have two options. Your first option is to set a storageClassName here for
```

File: pkg/apis/planetscale/v2/etcdlockserver_types.go [hunk 2]
```diff
@@ -82,9 +82,10 @@ type EtcdLockserverTemplate struct {
...
+	// a StorageClass that's configured to only provision volumes in the Availability Zone that corresponds to the
+	// Vitess cell, or instead you may set `volumeBindingMode: WaitForFirstConsumer` in the StorageClass.
```
Comment (avaidyanatha) [inline pkg/apis/planetscale/v2/etcdlockserver_types.go:87]: This just makes it slightly easier to read.

```suggestion
	// Vitess cell. Alternatively, you may set `volumeBindingMode: WaitForFirstConsumer` in the StorageClass.
```

**Review by avaidyanatha — COMMENTED** [review]
Two comments, looks good otherwise.

**Comment by enisoc** [comment]
I've pushed a comment to simplify this to only mentioning the new way. I'm not aware of any reason to use the old way except if you're on a super old k8s version, which we don't support anymore. I also applied the change to the tablet pool docs as well.

**Review by enisoc — APPROVED** [review]

**Pull request merged**